src/DotNetWorker/IFunctionActivator.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Microsoft.Azure.Functions.Worker
 {
-    public interface IFunctionActivator
+    internal interface IFunctionActivator
```

Comment:
Do we expect this to change? Should it stay public?

Reply:
I'd rather keep it internal by default and open it up in a follow-up.

Reply:
Not opposed to that, but this is one of the key interfaces that I can see people needing to use